frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-treatment-status/radiogram-card-content-treatment-status.component.ts

```diff
@@ -0,0 +1,28 @@
+import type { OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type { TreatmentStatusRadiogram } from 'digital-fuesim-manv-shared';
+import { UUID } from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import type { AppState } from 'src/app/state/app.state';
+import { createSelectRadiogram } from 'src/app/state/application/selectors/exercise.selectors';
+
+@Component({
+    selector: 'app-radiogram-card-content-treatment-status',
+    templateUrl: './radiogram-card-content-treatment-status.component.html',
+    styleUrls: ['./radiogram-card-content-treatment-status.component.scss'],
+})
+export class RadiogramCardContentTreatmentStatusComponent implements OnInit {
+    @Input() radiogramId!: UUID;
+
+    radiogram$!: Observable<TreatmentStatusRadiogram>;
+
+    constructor(private readonly store: Store<AppState>) {}
+
+    ngOnInit(): void {
+        const selectRadiogram = createSelectRadiogram(this.radiogramId);
+        this.radiogram$ = this.store.select(
+            selectRadiogram
+        ) as Observable<TreatmentStatusRadiogram>;
```

Comment:
What do you mean by 'like this'?

Reply:
The way I understood this is that there is no need to store the selector in a variable since it is only used in line 25 and no other place in the code.

Reply:
Done